source/_components/switch.toon.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
-title: "Toon"
-description: "Instructions how to integrate Toon Climate devices within Home Assistant."
-date: 2017-10-21 12:00
+title: "Toon Smart Plugs"
+description: "Instructions how to integrate Toon Smart Plugs within Home Assistant."
```

Comment:
`Smart Plugs` is not part of the product name and should be written in lower caps.